tsconfig.json

```diff
@@ -5,7 +5,8 @@
 		"target": "es2018", // Node.js 10
 		"lib": [
 			"es2018"
-		]
+		],
+		"sourceMap": true
```

Comment:
Oops. This should/could be removed - I added it so i could jump to the failing tests a wee bit quicker. I could use `tsc --sourceMap` in the future.